frontend/src/components/EasyModal/EasyModalHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import RightCloseButton from "../RightCloseButton";
+
+export function EasyModalHeaader({
```

Comment:
```suggestion
export function EasyModalHeader({
```

Rette skrivefeil for å unngå fremtidige merkelige bugs 😅

Reply:
Nice! Pusher fix nå :)